source/_docs/locked_out.md

```diff
@@ -9,7 +9,7 @@ or need to recover your data.
 ## Forgot user name
 
 If you’ve forgotten your username, ask the owner to help you.
-If you are the owner and have forgotten your user name, then you need to [prepare the system to start a new onboarding process](/docs/locked_out/#to-prepare-the-system-to-start-a-new-onboarding-process).
+If you are the owner and have access to the Home Assistant server, you can connect a terminal and [use the command line to list all user names](/common-tasks/os/#listing-all-users-from-the-command-line).
```

Comment:
I'd suggest to document the command in this context directly. We document `auth reset` below too, so the users list is kinda complementary to that.

Note that on the system terminal, there is a `ha >` promt already. This makes the initial `ha` unnecessary.